### PR TITLE
complete js-async-functions_sleep challenge

### PR DESCRIPTION
**Pull request by spacedoe**
# Context

This PR aims to tackle [The js-async-functions_sleep challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-async-functions/challenges-js-async-functions.mdjs).

# Changes

- [ ] Added functionality to the buttons.
- [ ] Added dark mode.

### CODE REVIEW

File: javascript/js-async-functions/js-async-functions_sleep/.eslintrc.json
Comment (nagimtm) [inline javascript/js-async-functions/js-async-functions_sleep/.eslintrc.json:1]: Solution is perfect, keep in mind please : ) After downloading challenges in your local machine you need initial commit that helps to compare  what you added or changed.

**Review by nagimtm — APPROVED** [review]

File: javascript/js-async-functions/js-async-functions_sleep/.eslintrc.json
Comment (spacedoe) [reply javascript/js-async-functions/js-async-functions_sleep/.eslintrc.json:1]: Thank you, Nagim!

Ah, I see... I did initial commit to the main branch, does it mean I need to do it twice to the main and the challenge branch?

Comment (nagimtm) [reply javascript/js-async-functions/js-async-functions_sleep/.eslintrc.json:1]: sorry Anna I forgot typing "push". After initial commit we need to push main branch on git hub :) then to create new branch and work on it.

Comment (spacedoe) [reply javascript/js-async-functions/js-async-functions_sleep/.eslintrc.json:1]: I think this is exactly what i did, I added the challenges, then added them with initial commit and pushed to the main, not sure why it doesn't show correctly... hmm...

**Pull request merged**